server/src/main/java/org/opensearch/index/TieredMergePolicyProvider.java

```diff
@@ -136,7 +136,7 @@ public final class TieredMergePolicyProvider implements MergePolicyProvider {
     private final boolean mergesEnabled;
 
     public static final double DEFAULT_EXPUNGE_DELETES_ALLOWED = 10d;
-    public static final ByteSizeValue DEFAULT_FLOOR_SEGMENT = new ByteSizeValue(2, ByteSizeUnit.MB);
+    public static final ByteSizeValue DEFAULT_FLOOR_SEGMENT = new ByteSizeValue(16, ByteSizeUnit.MB);
```

Comment:
Can you please add a comment just above this line explaining why we're not using the Lucene 10.x defaults?